src/App.jsx

```diff
@@ -6,22 +6,53 @@ import './App.css'
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
+function App() {
 
-const useSemiPersistentState = () => {
-  const [todoList,setTodoList] = useState(JSON.parse(localStorage.getItem('savedTodoList')) || []);
+  const [todoList,setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-    localStorage.setItem('savedTodoList', JSON.stringify(todoList))
-  }, [todoList])
 
-  return [todoList, setTodoList]
+  const fetchData = async() => {
+    const options = {
+      method: "GET",
+      headers: {
+        Authorization:`Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`
+      }
+    }
+    let url = `https://api.airtable.com/v0/${import.meta.env.VITE_AIRTABLE_BASE_ID}/${import.meta.env.VITE_TABLE_NAME}`
 
-}
+    try {
+      const response = await fetch(url, options)
 
-function App() {
-  const [count, setCount] = useState(0)
+      if (!response.ok) {
+        throw new Error(`An error occurred: ${response.status}`)
+      }
+
+      const data = await response.json()
+
+      const todos = data.records.map((todo) => {
+        return {id: todo.id, title: todo.fields.Title}
```

Comment:
Hi Carlos! 
Founded small issue here: 

The page does not show the todo title from the Airtable:
<img width="595" alt="image" src="https://github.com/user-attachments/assets/e49b1b3f-0ac0-42e0-b0c4-71acac269e80">
please look carefully on the data you received in your fetch function 
<img width="557" alt="image" src="https://github.com/user-attachments/assets/b94eab46-012a-4978-9cb6-3f53db677f3b">

<img width="427" alt="image" src="https://github.com/user-attachments/assets/0675f799-d5be-44f5-91a7-84e72daf1cdb">

keep in mind that all data is case sensitive